packages/clerk-js/src/index.browser.ts

```diff
@@ -15,7 +15,18 @@ const frontendApi =
   window.__clerk_frontend_api ||
   '';
 
-window.Clerk = new Clerk(publishableKey || frontendApi);
+const proxyUrl =
+  document.querySelector('script[data-clerk-proxy-url]')?.getAttribute('data-clerk-proxy-url') ||
+  window.__clerk_proxy_url ||
+  '';
+
+if (publishableKey && frontendApi) {
+  console.warn('Both publishableKey and frontendApi are set. Using publishableKey.');
```

Comment:
💡It is allowed to set both publishableKey and frontendApi and publishableKey has priority if both are set. Should we warn the developers about it ?

Reply:
These properties are mutually exclusive for now. ClerkJS constructor accepts both in the same argument, so the warning is not necessary.